test/bigtest/tests/createContact.e2e.js

```diff
@@ -0,0 +1,110 @@
+import { beforeEach, describe, it } from '@bigtest/mocha';
+import { expect } from 'chai';
+
+import setupApplication from '../helpers/setup-application';
+
+import { CreateContactInteractor } from '../interactors';
+
+describe('Create contact', () => {
+  setupApplication();
+
+  const createContact = new CreateContactInteractor();
+
+  beforeEach(function () {
+    const organization = this.server.create('organization');
+
+    return this.visit(`/organizations/${organization.id}/contacts/add-contact`, () => {
+      expect(createContact.$root).to.exist;
+    });
+  });
+
+  describe('Create contact form', () => {
+    it('create contact form should be present', () => {
+      expect(createContact.saveButton.isPresent).to.be.true;
+      expect(createContact.addressForm.addresses().length).to.be.equal(0);
+      expect(createContact.phoneForm.phones().length).to.be.equal(0);
+      expect(createContact.emailForm.emails().length).to.be.equal(0);
+      expect(createContact.urlForm.urls().length).to.be.equal(0);
+    });
+
+    describe('Add address', () => {
```

Comment:
I'm not sure it should be nested in `describe('Create contact form'`.

Reply:
Fixed